pcdet/utils/stats_utils.py

```diff
@@ -136,6 +136,8 @@ def compute(self):
             results = {}
             for mname in self.metrics_name:
                 mstate = getattr(self, mname)
+                if not isinstance(mstate, (list, tuple)):
```

Comment:
@danish87 do you remember which state wasn't the `list`?

Reply:
It was not list when i tested

Reply:
Yeah, I know. I mean for which state it wasn't?

Reply:
In the distributed setting, I think the `mstate` being returned in #138 (above code snippet) would be a `generator` instead of a `list`. This is because in distributed setting, TorchMetrics synchronizes the states between processes using `sync_context()` which returns the synced states as a `generator`.